packages/.yarnrc.yml

```diff
@@ -0,0 +1,14 @@
+npmPublishAccess: public
+initScope: hashicorp
+initFields:
+  license: MPL-2.0
+  version: 0.0.1
+  author: FIXME
+  repository:
+    type: git
+    url: "git+https://github.com/hashicorp/structure.git"
+  bugs:
+    url: "https://github.com/hashicorp/structure/issues"
+  homepage: "https://github.com/hashicorp/structure#readme"
+  publishConfig:
+    access: public
```

Comment:
This is added in conjunction with `npmPublishAccess` to ensure that no matter which package manager we use to publish, it'll be published with the correct access.